tableauserverclient/server/request_factory.py

```diff
@@ -480,8 +480,9 @@ def update_req(self, workbook_item):
         if workbook_item.owner_id:
             owner_element = ET.SubElement(workbook_element, 'owner')
             owner_element.attrib['id'] = workbook_item.owner_id
-        if workbook_item.materialized_views_config['materialized_views_enabled'] \
-                and workbook_item.materialized_views_config['run_materialization_now']:
+        if workbook_item.materialized_views_config is not None and \
```

Comment:
Do both 'materialized_views_enabled' and 'run_materialization_now' fields have to be present to be able to update MaterializedViewsEnablementConfig for workbooks?

Reply:
Thanks for pointing it out! Changed "run_materialization_now" to optional which is consistent with the server side code.